public/app/features/plugins/admin/pages/PluginDetails.tsx

```diff
@@ -51,19 +51,27 @@ export default function PluginDetails({ match }: PluginDetailsProps): JSX.Elemen
             <PluginLogo
               src={plugin.info.logos.small}
               className={css`
-                object-fit: cover;
+                object-fit: contain;
                 width: 100%;
                 height: 68px;
                 max-width: 68px;
               `}
             />
 
             <div className={styles.headerWrapper}>
-              <h1>{plugin.name}</h1>
+              <h1 className={styles.header}>
+                <a
+                  className={css`
+                    text-decoration: underline;
+                  `}
+                  href={'/plugins'}
+                >
+                  Plugins
```

Comment:
I think breadcrumb functionality is built into the `Page` component (navModel). Any specific reason why that approach isn't used here?

Reply:
Thanks @tskarhed, I wasn't aware of that! Right now we're still in the process of migrating this code into core. I'll put this PR on ice and look into replacing the custom page and tab stuff with the `Page` component. Not sure how much work it is to migrate to using these components and the navmodel.

Reply:
There is a risk that you lose some customizability when using it, so it might not be appropriate.

Reply:
Unfortunately due to the design of the header on this Plugin Details page I cannot easily use the Page components due to the lack of customisation as you noted @tskarhed.